homeassistant/components/tts/strings.json

```diff
@@ -0,0 +1,60 @@
+{
+  "services": {
+    "say": {
+      "name": "Say a TTS message",
+      "description": "Says something using text-to-speech on a media player.",
+      "fields": {
+        "entity_id": {
+          "name": "Entity",
+          "description": "Name(s) of media player entities."
+        },
+        "message": {
+          "name": "Message",
+          "description": "Text to speak on devices."
+        },
+        "cache": {
+          "name": "Cache",
+          "description": "Control file cache of this message."
```

Comment:
```suggestion
          "description": "Stores this message locally so that when the text is requested again, the output can be  produced more quickly."
```